frontend/src/app/insight/page.tsx

```diff
@@ -0,0 +1,116 @@
+import "globals";
+import Link from "next/link";
+import { MaterialSymbol } from "react-material-symbols";
+import { Button } from "@/components/ui/button";
+
+export default function Home() {
+  return (
+    <div className="grid h-full flex-1 grid-cols-2">
+      <div className="grid h-full content-center rounded-tr-2xl bg-background">
+        <div className="row-auto grid content-center space-y-12 p-24">
+          <p className="text-6xl font-extrabold">
+            Welcome back, <br></br>John!
+          </p>
+          <p className="text-200 text-200 text-4xl font-medium">
+            Here’s your summary.
+          </p>
+
+          <div className="relative rounded-xl p-4">
+            <div className="absolute inset-0 animate-pulse rounded-xl border-4 border-secondary delay-100"></div>
+            <div className="flex">
+              <div className="flex-1 pr-39 relative z-10 flex gap-x-2">
+                <MaterialSymbol icon="bolt" size={32}></MaterialSymbol>
+                <p className="text-[1.2vw] font-medium">Electricity</p>
```

Comment:
Why are you using `vw` for text size? I'm not saying it's wrong, I'm just curious on why you're not using `text-lg`, `text-xl` and etc

There are more lines with this issue, I'm just making a comment on the first

Reply:
You're right, the size supposedly should change based on the window's width, but in this case is really not necessary.